root/programs/C#/Frameworks/Infrastructure/ServiceInterface/ASPNETWebService/ASPNETWebService/Controllers/JsonController.cs

```diff
@@ -0,0 +1,907 @@
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#endregion
+
+//**********************************************************************************
+//* クラス名        ：JsonController
+//* クラス日本語名  ：ASP.NET WebAPI JSON-RPCの個別Webメソッドを公開するサービス インターフェイス基盤。
+//*
+//* 作成日時        ：－
+//* 作成者          ：生技
+//* 更新履歴        ：
+//* 
+//*  日時        更新者            内容
+//*  ----------  ----------------  -------------------------------------------------
+//*  2017/08/18  西野 大介         新規作成
+//**********************************************************************************
+
+using System;
+using System.Xml;
+using System.Data;
+using System.Reflection;
+using System.Web;
+using System.Web.Http;
+using System.Web.Http.Cors;
+using System.Net;
+using System.Net.Http;
+using System.Collections.Generic;
+using System.Runtime.ExceptionServices;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using Touryo.Infrastructure.Business.Presentation;
+using Touryo.Infrastructure.Business.Common;
+using Touryo.Infrastructure.Business.Util;
+using Touryo.Infrastructure.Framework.Common;
+using Touryo.Infrastructure.Framework.Transmission;
+using Touryo.Infrastructure.Framework.Exceptions;
+using Touryo.Infrastructure.Framework.Util;
+using Touryo.Infrastructure.Public.Db;
+using Touryo.Infrastructure.Public.Log;
+using Touryo.Infrastructure.Public.Util;
+
+using WSIFType_sample;
+using WSServer_sample.Common;
+
+namespace ASPNETWebService.Controllers
+{
+    /// <summary>
+    /// ASP.NET WebAPI JSON-RPCの個別Webメソッドを公開するサービス インターフェイス基盤。
+    /// </summary>
+    [EnableCors(
+        // リソースへのアクセスを許可されている発生元
+        origins: "*",
+        // リソースによってサポートされているヘッダー
+        headers: "*",
+        // リソースによってサポートされているメソッド
+        methods: "*",
+        // 
+        SupportsCredentials = true)]
+    [MyBaseApiController()]
+    public class JsonController : ApiController
+    {
+        #region テスト用
+
+        /// <summary>test</summary>
+        /// <returns>string</returns>
+        [HttpGet]
+        public string test()
+        {
+            return "test";
+        }
+
+        #endregion
+
+        #region グローバル変数
+
+        /// <summary>インプロセス呼び出しの名前解決シングルトン クラス</summary>
+        /// <remarks>
+        /// 初期化は起動時の１回のみであり、
+        /// 読み取り専用のデータを保持する場合
+        /// のみに適用するデザインパターンとする。
+        /// </remarks>
+        private static InProcessNameService IPR_NS = new InProcessNameService();
+
+        #endregion
+
+        #region コンストラクタ
+
+        /// <summary>constructor</summary>
+        public JsonController()
+        {
+        }
+
+        #endregion
+
+        #region ASP.NET WebAPI JSON-RPCのWebメソッド
+
+        #region 個別部
```

Comment:
@daisukenishino 

I think that this file is the foundation class when using json and web api.
So, I think that the methods specific to tasks are unnecessary in this class.